DEV-3/DEV-3/EntryPoint.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+
+namespace DEV_3
+{
+    /// <summary>
+    /// It is entry point of the program and it is print a number in the selected number system
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            if (Int32.Parse(args[1]) >= 2 && Int32.Parse(args[1]) <= 20)
+            {
+                NumbersConversion enteredValues = new NumbersConversion(Int32.Parse(args[0]), Int32.Parse(args[1]));
+                foreach (var convertedNumber in enteredValues.ConversionToRadix())

```

Comment:
Very bad decision to output resulting string. Compose the string from list as output it by one call of WriteLine